packages/best-practices/package.json

```diff
@@ -47,12 +47,12 @@
     "@typespec/compiler": "workspace:~0.48.1",
     "@typespec/eslint-config-typespec": "workspace:~0.48.0",
     "@typespec/eslint-plugin": "workspace:~0.48.0",
-    "eslint": "^8.42.0",
```

Comment:
Why are we upgrading the base versions of packages?  In `azure-sdk-for-js`, we only update the base when there is a new major, or when we know we need a feature or bug fix in a specific minor or patch.

Reply:
why not? This should only matter for `peerDependencies`. Just running `rush upgrade-interactive` having to filter out things makes it much harder to figure out what is out of date. not updating it can always lead to it being downgraded locally which happened in the past with docusaurus for example.

Reply:
Discussed offline.  I'm fine with maintaining the approach TypeSpec has been using.